contracts/SEuroOffering.sol

```diff
@@ -64,8 +64,9 @@ contract SEuroOffering is Ownable {
     /// @param _token byte array value for the token that you'd like to estimate the exchange value for
     /// @param _amount the amount of the given token that you'd like to estimate the exchange value for
     function readOnlyCalculateSwap(bytes32 _token, uint256 _amount) external view returns (uint256) {
+        if (_token == bytes32("ETH")) _token = bytes32("WETH");
```

Comment:
are you doing this replacement due to the setup on the frontend?

Reply:
not sure if i've missed any discussions on that _front_ that is, pun not intended

Reply:
just provides a way to use `ETH` when estimating prices, given `ETH` isn't an accepted token in token manager. open to feedback on that though

Reply:
ah, I see now. it's a fair way to get the ETH price, assuming that it (WETH) tracks its price "close enough", **as long as** there is no other better data feed for ETH.

Reply:
all the `weth` data feeds are `eth` data feeds. chainlink doesn't have any `weth` data feeds. because you can always swap an exact amount of eth for weth, the prices will always be identical i guess (and if not then i would guess that's the end of weth). so i think we're good

incidentally i think there's efforts being made to allow eth to follow erc20 protocol. one day ...

Reply:
ok, that's good. i was just looking at <insert your favourite coin tracker> and saw that ETH and WETH differed by a few $'s.